src/d3a/constants.py

```diff
@@ -44,3 +44,5 @@
 SIMULATION_PAUSE_TIMEOUT = 600
 
 D3A_TEST_RUN = False
+
+IS_CANARY_NETWORK = True
```

Comment:
IMO the default value should be `False` for local runs. Or am misunderstanding?

Reply:
Remnant from testing, changing this, thanks